content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -182,7 +182,7 @@ name=Kubernetes
 baseurl=https://packages.cloud.google.com/yum/repos/kubernetes-el7-\$basearch
 enabled=1
 gpgcheck=1
-repo_gpgcheck=1
+repo_gpgcheck=0
```

Comment:
I think we can just delete this line, and let the default of 0 take effect, which also has less appearance of endorsing disabling it. https://linux.die.net/man/5/yum.conf

Reply:
@BenTheElder I checked using `centos:7` Docker image, and confirm that it works with this line deleted.
I will reflect your suggestion. 😊

Reply:
Done! PTAL again 😊